common/constants.go

```diff
@@ -107,3 +107,8 @@ const (
 	// below which a warning will be logged
 	CriticalLongPollTimeout = time.Second * 20
 )
+
+const (
+	// DefaultTransactionSizeLimit is the largest allowed transaction size to persistence
+	DefaultTransactionSizeLimit = 14 * 1024 * 1024
```

Comment:
consider 5 MB?

Reply:
@samarabbas What do you think about this? My understanding was the limit on cassandra and mysql was more than 14MB. But looking at this https://docs.datastax.com/en/cql/3.3/cql/cql_reference/refLimits.html it looks like the limit might be lower?

Reply:
Currently the size of each transaction is bounded by cassandra config value of 'max_mutation_size_in_kb'.  The default value is 16MB for this config knob.  I think we should allow transactions size to go as much as the allowed limit by persistence choice.  I think 14MB makes sense as a default.